ios/Peepz/Peepz/Gallery/GalleryItemView.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+
+struct GalleryItemViewState: Hashable {
+    let imageName: String?
+    let location: String?
+    let name: String
+    let isActive: Bool
+}
+
+struct GalleryItemView: View {
+    let state: GalleryItemViewState
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            AsyncImage(
```

Comment:
Im not too happy about this solution for downloading pictures, but it works for this iteration.